<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <widget class="QLabel" name="l_controller">
           <property name="text">
            <string>Контроллер</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="controller"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="l_module">
           <property name="text">
            <string>FTA/RFTA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fta_rfta"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="l_pipe">
           <property name="text">
            <string>SCM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="scm">
           <item>
            <property name="text">
             <string>CM-1-536</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Канал</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pipe"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="l_name">
       <property name="text">
        <string>Название сигнала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="l_cipher">
           <property name="text">
            <string>Шифр</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cipher">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="l_type">
           <property name="text">
            <string>Тип сигнала</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="type">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="l_min">
           <property name="text">
            <string>Минимум</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="min"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="l_max">
           <property name="text">
            <string>Максимум</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="max"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <widget class="QLabel" name="l_unit">
           <property name="text">
            <string>Ед. изм.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="unit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="l_adress">
       <property name="text">
        <string>ТЭГ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tag"/>
     </item>
     <item>
      <widget class="QLabel" name="l_points">
       <property name="text">
        <string>Точки калибровки:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="l_point1">
           <property name="text">
            <string>№1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="point1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QLabel" name="l_poin3">
           <property name="text">
            <string>№2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="point2">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <widget class="QLabel" name="l_point2">
           <property name="text">
            <string>№3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="point3">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="l_point4">
           <property name="text">
            <string>№4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="point4">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Примечание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="note"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Применить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
